src/org/aswing/JTabbedPane.hx

```diff
@@ -66,7 +67,23 @@ class JTabbedPane extends AbstractTabbedPane{
     private var _leadingOffset: Int;
     private function get_leadingOffset(): Int { return getLeadingOffset(); }
     private function set_leadingOffset(v: Int): Int { setLeadingOffset(v); return v; }
-	
+
+	public var tabs(get, set):Array<TabInfo>;
+	private var _tabs:Array<TabInfo>;
+	private function get_tabs():Array<TabInfo> { return getTabs(); }
+	private function set_tabs(v:Array<TabInfo>):Array<TabInfo> { setTabs(v); return v; }
+
+	public function getTabs():Array<TabInfo> {
```

Comment:
It's better to move the content of methods getTabs and setTabs to get_tabs and set_tabs directly.